test/services/helpdesk/index.test.js

```diff
@@ -89,5 +94,29 @@ describe('helpdesk service', function() {
 			}
 		);
 	});
+
+	it('DELETE /helpdesk with user from same school', () => {
+
+		helpdeskService.remove('5836bb5664582c35df3bc214', { payload: {userId: '0000d213816abba584714c0a'}})
+			.then(result => {
+				expect(result).to.not.be.undefined;
+			}
+		);
+
+	});
+
+	it('DELETE /helpdesk with user from different school', () => {
+
+		helpdeskService.remove('5836bb5664582c35df3bc214', { payload: {userId: '599ec1688e4e364ec18ff46e'}})
```

Comment:
Gleiche Problem abermals ... hier versucht du sogar etwas zu löschen was schon nicht mehr existiert und dann kommt natürlich ein 404er, aber was du ja eigentlich erreichen willst ist ein 403er... aber wieso ist es schief gelaufen? Weil er wieder in Zeile 206/207 springt...